cabal.project

```diff
@@ -56,9 +56,4 @@ allow-newer:
 -- see https://github.com/haskellari/postgresql-simple/issues/104
     postgresql-simple:base
   , postgresql-simple:template-haskell
--- https://github.com/haskell-servant/servant-js/pull/49
-  , servant-js:base
-  , servant-js:lens
-  , servant-js:text
--- https://github.com/haskell-servant/servant-multipart/pull/64
-  , servant-multipart:lens
+  , swagger2:base
```

Comment:
did you submit a PR there?

Reply:
I've made a PR now: https://github.com/GetShopTV/swagger2/pull/244
Link added in1c2299be62abc0c48997f9bed0157635dfb50202

Reply:
It got released: https://hackage.haskell.org/package/swagger2-2.8.6

Reply:
Override removed in 91269730729a39d757d111e311b0490029b16b40